game_eggs/eft/README.md

```diff
@@ -0,0 +1,51 @@
+# Escape from Tarkov
+
+# Singleplayer Tarkov Server
+
+## After server installation
+
+Run the server once to generate the necessary config files for the coop mod, then edit `coopConfig.json` in `/home/container/user/mods/SITCoop/config` with your external IPv4.
+
+## Client for connecting to the server
+
+[Stay in Tarkov](https://github.com/stayintarkov/StayInTarkov.Client)
+
+An Escape From Tarkov BepInEx module designed to be used with the SIT.Aki-Server-Mod with the ultimate goal of "Offline" Coop
+
+Installation instructions are provided in [the project's README.md file](https://github.com/stayintarkov/StayInTarkov.Client#client-install).
+
+**2024-02-05 Note, please read**:
+> NOTE: As of 05/02/2024 the downgraders used by the manager and in the link provided for the manual install will not downgrade to the correct version and/or fail. Please check the SIT Discord for the latest EFT downgrader
+
+What this means: the user needs to manually downgrade EFT's version with a patch matching the live game's version from ([the SIT discord linked in the SIT repo](https://github.com/stayintarkov/StayInTarkov.Client#support)) in the #downgrade-eft channel.
+
+To install the SIT client:
+- Install the live Escape from Tarkov game from the official launcher.
```

Comment:
Should also add that they need to launch the game once and be at the main menu of the game then close it.